docs/_controls/ct-11.md

```diff
@@ -11,12 +11,22 @@ title: Human feedback loop
+  i) Prompt finetuning: Analyse the feedback from the users and review them against the answers provide by LLM. Based upon the analysis find the gaps in purpose of the solution, based upon reponses in context of the user feedback provided. Tune to prompt to bridge those gaps. 
+ ii) RAG document update: Analyze the low rated responses and look to content improvement opportunities wherever quality of the content is the root cause.
+iii) Model/Data Drift: Feedback data should be able quantifiably detect the model drift or data drift due to changes in foundational model version or data quality over a period of time. 
+ iv) Other future potential advanced uses i.e. Reinforcement Learning or Finetuning of the model itself.
+Defining these goals of the data usage will make ensure the longterm viability of the solution.
+3. User Experience: The team implementing the solution should survey their target audience on how open they are to provide feedback and whether the feedback will hamper the effectiveness of the solution in any way.
+4. Wide vs Narrow Feedback: Sometimes user experience is very impoprtant to solution and the feedback can come in the way of user experince. In such scenarios, the team shoud consider created a smaller group tester SMEs within the users who can collaborate with development team to provide continuous elaborate feedback to improve the solution.
+    
+As stated earlier, there are several ways one can collect the feedback. The choices are very much dependent on the use case and intention of data usage. Following are the two major categories of feedback: 
 
-There are several ways one collect the feedback. 
+1. Quantifiable Feedback: These are generally questions to the user which can be answered with categorical or numerical rating. For e.g. How you rate the repose of the chatbot on scale 1-5 with descripting of scaling parameters. Qauntifiable parameters can be potentiatlly used in conjuction with other metrics for KPI. 
+2. Qaulitative Feedback: These are open eneded quations where the use is expected to write free form text as feedback. This allow user to provide data points not covered in Quantifiable feedback. One can use NLP or another LLM to analyze and derive insights from it.
 
-
-- Allow people to rate or report answers
+Another aspect of Human Feeback which one should consider is related to the growing field of LLM-as-a-Judge (CT-15). When we use LLM-as-a-Judge, one should take the same Quantifiable and Qaulitative feedback from the LLM as they will do from humans. This will provide them an opportunity to compare feedback between machine and humans. Also it will allow to rate the effectiveness of LLM-as-a-Judge. One should consider doing Narrow feedback on atleast a sample of the LLM-as-a-Judge results. The sample size (in terms of %) and the methoidology is very use case dependent. For e.g. if the use case is very critical them one needs to verify higher number of samples. 
+    
```

Comment:
- Feedback Frequency and Timing: Determine optimal frequency and timing for collecting feedback, including real-time feedback, periodic surveys, or a combination of both.

- Feedback Prioritization: Develop a method for prioritizing feedback based on its potential impact on model performance or alignment with business objectives. Automate feedback analysis for identifying trends, insights, priority  more efficiently.
- Feedback Loop Closure: Communicate improvements back to users based on their feedback to encourage continued engagement in providing input.
- Ethical Considerations: Address ethical concerns related to collecting and using human feedback, including privacy issues and potential biases in feedback collection methods.
- Multi-modal Feedback: Consider collecting feedback across different modalities (e.g., text, voice, visual) for more comprehensive insights into user experiences.
- Transparency into feedback Integration Process: Provide detailed steps on how collected feedback will be integrated into AI models, including necessary review or approval processes before implementation.
- Measuring Feedback Impact and effectiveness: Measure  how implemented feedback affects model performance over time, ensuring continuous improvement based on human  input to measure effectiveness of feedback collection process based on user engagement and its impact on model performance.